test/widget_tests/utils/popup_manager/popup_manager_test.dart

```diff
@@ -0,0 +1,631 @@
+import 'dart:async';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_core/flutter_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('Popup Manager Tests', () {
+    testWidgets(
+      'Loader Test',
+      (tester) async {
+        final navigatorKey = GlobalKey<NavigatorState>();
```

Comment:
The navigatorKey and popupManager values can be set up within a group in the setup method to eliminate code repetition.

Reply:
Fixed